src/smc-webapp/chat/store.ts

```diff
@@ -4,6 +4,14 @@ import * as immutable from "immutable";
 // Internal Libraries
 import { Store } from "../app-framework/Store";
 
+export type MentionList = immutable.List<{
```

Comment:
Oh, I didn't realize you could do that with immutable.List and typescript.  Good to know!